### PR TITLE
Adjust IRadarSignature to support per-cell colors.

### PR DESCRIPTION
**Pull request by pchote**
For use primarily by the new map editor.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/AppearsOnRadar.cs [hunk 1]
```diff
@@ -12,6 +12,7 @@
 using System.Drawing;
 using System.Linq;
 using OpenRA.Traits;
+using OpenRA.Primitives;
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/AppearsOnRadar.cs]: [SA1210] Using directives must be sorted alphabetically by the namespaces.

**Comment by Mailaender** [comment]
Can't spot any regressions. :white_check_mark:

**Comment by obrakmann** [comment]
Me neither, code looks good, too. :+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/b1ffeb667c28e2be24a3d70572deb8b6fb01d297%5E...b1ffeb667c28e2be24a3d70572deb8b6fb01d297)

**Pull request merged**